src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -48,7 +48,7 @@ internal partial class DiagnosticIncrementalAnalyzer : IIncrementalAnalyzer
             int correlationId,
             Workspace workspace,
             DiagnosticAnalyzerInfoCache analyzerInfoCache,
-            AbstractHostDiagnosticUpdateSource hostDiagnosticUpdateSource)
+            AbstractHostDiagnosticUpdateSource? hostDiagnosticUpdateSource)
         {
             Contract.ThrowIfNull(analyzerService);
```

Comment:
is this necessary?

Reply:
Maybe we remove them after we finish enabling nullable reference types, but for now it doesn't seem to hurt.